src/components/dashboard/FaceVerification/screens/ErrorScreen.js

```diff
@@ -20,7 +20,12 @@ const ErrorScreen = ({ styles, screenProps }) => {
   const kindOfTheIssue = get(exception, 'name')
   const isGeneralError = !kindOfTheIssue || !(kindOfTheIssue in ErrorScreen.kindOfTheIssue)
 
-  const [verificationAttempts, trackNewAttempt, resetAttempts] = useVerificationAttempts()
+  const [
```

Comment:
@yaroslav-fedyshyn-nordwhale let's change a bit hook interface to: 

```typescript
{
  attempts: Array<Error>; // attempts history, former verificationAttemptErrMessages but containing error objects themselves instead of message strings
  attemptsCount: number; // former verificationAttempts
  track(exception: Error); // former trackNewAttempt but pass exception itself instead of just message string
  reset(); // former resetAttempts
}
```